src/Components/Application/Application.jsx

```diff
@@ -59,37 +60,74 @@ const ApplicationSelector = styled.div`
   }
 `;
 
-function Application({ applications, newReview, history, match, user }) {
-  const appId = match.params.organizationId;
-
-  const [loadedReview] = usePromise(GET.getReviewAPI, {
-    user,
-    applicationId: appId
-  });
+function Application({ applications, newReview, history, match, user, program }) {
 
+  const appId = match.params.organizationId;
   const [review, dispatchReviewUpdate] = useReducer(reviewReducer, null);
   const isRated = useRef(false);
 
-  // Load the fetched review
+  // const [loadedReview, refetch] = usePromise(GET.getReviewAPI, {
+  //   user,
+  //   applicationId: appId
+  // });
+  //
+  // useEffect(() => {
+  //   refetch({user, applicationId: appId});
+  // }, [])
+
```

Comment:
@xinhaoz I commented out the new changes using usePromise for now. I couldn't get the review to load when clicking open application.